script/c511027014.lua

```diff
@@ -0,0 +1,71 @@
+--ハイドライブ・リ・アドベント
+--Hydradrive Readvent
+--Scripted by Playmaker 772211
+local s,id=GetID()
+function s.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetTarget(s.target)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+end
+function s.cfilter(c)
+	return c:IsFaceup()
+end
+function s.filter(c,e,tp,atk)
+	return c:IsType(TYPE_LINK) and c:IsAttackBelow(atk) and c:IsCanBeSpecialSummoned(e,0,tp,false,false)

```

Comment:
"with less ATK than the monster your opponent controls"
`IsAttackBelow` check if the Attack is Equal or Below the parameter
so the filter should be
```suggestion
	return c:IsType(TYPE_LINK) and c:GetAttack()<atk and c:IsCanBeSpecialSummoned(e,0,tp,false,false)
```
__but only if `atk` is an integer containing the greatest attack__, else you should make the appropriate change like so
```suggestion
	return c:IsType(TYPE_LINK) and c:GetAttack()<tg:GetFirst():GetAttack() and c:IsCanBeSpecialSummoned(e,0,tp,false,false)
```
But for this i'm not sure (tested with the second one with my version of the script and seems to work, should be tested :eyes:)